src/runner/http/response.py

```diff
@@ -153,9 +160,22 @@ def responseToFlatBuffers(self) -> flatbuffers.Builder:
             A6HTTPReqCallResp.AddAction(builder, stop)
             res = A6HTTPReqCallResp.End(builder)
             builder.Finish(res)
+        elif rpcType == RunnerHttpProtocol.RPC_TEST:
```

Comment:
I am afraid the RPC_TEST is not part of the protocol. You can't find it in APISIX. I only use it in the Go Plugin Runner for some dirty test job.